### PR TITLE
util.puts is deprecated, and stderr is useful

### PR DESCRIPTION
**Pull request by erikwoods**
I know what you said about this repo, but... I also know that it's still out there for others to use, and this is just a minor improvement you may want to merge in. maybe. :hamburger: 

EDIT: I also made a change to make use of `stderr`

PS: thanks to @kherrick for the tip about `stderr`

### CODE REVIEW

**Comment by erikwoods** [comment]
PS: thanks to @kherrick for the tip about `stderr`

File: lib/main.js [hunk 1]
```diff
@@ -2,11 +2,11 @@
 /*globals module require*/
 var exec = require('child_process').exec
   , fs   = require("fs")
-  , path = require("path")
-  , util = require("util");
+  , path = require("path");
 
-function puts(error, stdout/*, stderr*/) {
-  util.puts(stdout);
+function consolelog(error, stdout, stderr) {
```
Comment (kalisjoshua) [inline lib/main.js]: I don't think `consolelog` is necessary as a name. I would suggest just `log`.

File: lib/main.js [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+  console.log(stdout);
+  stderr ? console.log(stderr) : null;
```
Comment (kalisjoshua) [inline lib/main.js]: This ternary should be an `if` instead; that way there is no need for the `null` else block.

**Comment by kalisjoshua** [comment]
Other than the two changes I mention this looks like a good update after quite a while.

File: lib/main.js (hunk 1 above)
Comment (erikwoods) [reply lib/main.js]: I originally wanted to just name it `log`, but thought being verbose was safer. I'll change.

File: lib/main.js (hunk 2 above)
Comment (erikwoods) [reply lib/main.js]: Okey dokey.

**Pull request merged**